pkg/rke2/rke2.go

```diff
@@ -61,9 +62,8 @@ type ExtraEnv struct {
 
 // Valid CIS Profile versions
 const (
-	CISProfile15           = "cis-1.5"
-	CISProfile16           = "cis-1.6"
-	defaultAuditPolicyFile = "/etc/rancher/rke2/audit-policy.yaml"
+	CISProfile123          = "cis-1.23"
```

Comment:
Is the "cis-1.5" and "cis-1.6" removal related to PSA?

Reply:
yeah PSA changes will go in cis-1.23 and I think we agreed on deprecating and removing cis-1.5 and 1.6